go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/cloud-gov/cf-domain-broker-alb
+
+go 1.21.7
+
+require (
+	code.cloudfoundry.org/lager v1.0.1-0.20180322215153-25ee72f227fe
+	github.com/aws/aws-sdk-go v1.13.34
+	github.com/cloud-gov/cf-cdn-service-broker v0.0.0-20180402200151-f1f308020e14
+	github.com/cloudfoundry-community/go-cfclient v0.0.0-20180404210329-cfc053bbd960
```

Comment:
Is this the safe commit to pin to?

Reply:
As far as I can tell it should be a safe commit cause it's an old dev version of the code